types/react-native-highlight-words/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-native-highlight-words 1.0
+// Project: https://github.com/clauderic/react-native-highlight-words
+// Definitions by: Ivan Stelmakh <https://github.com/stelmakhivan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import { TextProps } from 'react-native';
+
+export interface HighlighterProps {
+    autoEscape?: boolean;
+    highlightStyle?: TextProps['style'];
+    sanitize?: () => void;
```

Comment:
```suggestion
    sanitize?: (text: string) => string
```
should it tread: https://github.com/clauderic/react-native-highlight-words#props